docs/networking.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Networking
+subtitle: Connecting ev3dev to the internet and other devices
+---
+
+If you want to access your ev3dev installation from a PC or install new software
+on your EV3, you'll need to connect to a network. Here are some resources to
+help you out.
+
+# Connecting to the internet
+
+You have a few options for connecting to the internet.
+
+### With a Wi-Fi dongle
+
+The simplest route is to purchase a Wi-Fi dongle and plug it into the EV3's USB
+port. Almost any USB Wi-Fi dongle which supports Linux will work, including the
```

Comment:
The "officially supported" ones are going to be the ones I actually own.

    0846:9030 NetGear, Inc. WNA1100 Wireless-N 150 [Atheros AR9271]
    7392:7811 Edimax Technology Co., Ltd EW-7811Un 802.11n Wireless Adapter [Realtek RTL8188CUS]
    0bda:8176 Realtek Semiconductor Corp. RTL8188CUS 802.11n WLAN Adapter

The first two are the ones officially supported by LEGO.

The last one is [this](http://www.edupwireless.com/product-1-2-5-mini-wireless-usb-adapter-en/137318) except mine has Dexter Industries branding. It works with the official EV3 firmware as well.